src/Features/DiagnosticsTestUtilities/CodeActions/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -70,6 +70,9 @@ public Test()
             [StringSyntax(PredefinedEmbeddedLanguageNames.CSharpTest)]
             public new string FixedCode { set => base.FixedCode = value; }
 
+            [StringSyntax(PredefinedEmbeddedLanguageNames.CSharpTest)]
+            public new string BatchFixedCode { set => base.BatchFixedCode = value; }
```

Comment:
@sharwell  to check on this.  curious why we haven't needed this yet.

Reply:
It's been around for a long time. It's needed for cases where Fix All in Document does not produce the same result as running the individual fixes in sequence.